src/app/static/src/app/store/translations/locales.ts

```diff
@@ -404,7 +404,7 @@ const en: Translations = {
     back: "back",
     both: "Both",
     browseADR: "Browse ADR",
-    bubble: "Bubble Plot",
+    bubble: "Bubble",
```

Comment:
User request? 😆

Reply:
Yeah, noticed this wasn't consistent with the other tabs. Not sure if came from Rachel originally or us..